browser/help/topics/v4-browser-hts.md

```diff
@@ -182,12 +182,15 @@ Row fields:
   - `xstart`: Transcript genomic start position (format: chromosomeposition).
   - `xstop`: Transcript genomic stop position (format: chromosomeposition).
   - `exons`: Array containing transcript exon information.
-  - `feature_type`: Exon type (e.g., CDS).
-  - `start`: Exon genomic start position (position only).
-  - `stop`: Exon genomic stop position (position only).
-  - `xstart`: Exon genomic start position (format: chromosomeposition).
-  - `xstop`: Exon genomic start position (format: chromosomeposition).
+    - `feature_type`: Exon type (e.g., CDS).
+    - `start`: Exon genomic start position (position only).
+    - `stop`: Exon genomic stop position (position only).
+    - `xstart`: Exon genomic start position (format: chromosomeposition).
+    - `xstop`: Exon genomic start position (format: chromosomeposition).
```

Comment:
Just calling out that this is not formatter/linter noise, these fields were previously at the wrong level of indentation, these should be nested under the `exons` on line 184 and now properly are.